product-service/handlers/createProduct.ts

```diff
@@ -0,0 +1,50 @@
+import {APIGatewayProxyHandler} from 'aws-lambda';
+import 'source-map-support/register';
+import {transaction} from "./pgClient";
+import {corsHeaders} from "./cors";
+
+export const createProduct: APIGatewayProxyHandler = async (event, _context) => {
+    if (!event.body) {
+        return {statusCode: 400, headers: corsHeaders, body: JSON.stringify({error: "Missing product data"}, null, 2)}
+    }
+    console.log(`createProduct, body=${event.body}`);
+    const reqData = JSON.parse(event.body);
+    try {
+        validate(reqData);
+    } catch (err) {
+        return {statusCode: 400, headers: corsHeaders, body: JSON.stringify({error: err}, null, 2)}
+    }
+
+    const {title, description, price, count} = reqData;
+    try {
+        const productId = await transaction(async client => {
+            const response = await client.query("INSERT INTO product (title, description, price) VALUES ($1, $2, $3) RETURNING id", [title, description, price]);
+            const productId = response.rows[0].id;
+            await client.query("INSERT INTO stock (product_id, count) VALUES ($1, $2)", [productId, count])
```

Comment:
It's would be nice to move all query-specific logic to DB client, so that logic in lamdas will be more clear.